doc/getstarted/basic_usage/index_en.rst

```diff
@@ -26,10 +26,10 @@ Suppose the true relationship can be characterized as ``Y = 2X + 0.3``, let's se
                 x = random.random()
                 yield [x], [2*x+0.3]
 
-Train a NeuralNetwork
+Train a Neural Network
 ----------------------
 
-To recover this relationship between ``X`` and ``Y``, we use a neural network with one layer of linear activation units and a square error cost layer. Don't worry if you are not familiar with these terminologies, it's just saying that we are starting from a random line ``Y' = wX + b`` , then we gradually adapt ``w`` and ``b`` to minimize the difference between ``Y'`` and ``Y``. Here is what it looks like in PaddlePaddle:
+To recover this relationship between :math:`X` and :math:`Y`, we use a neural network with one layer of linear activation units and a square error cost layer. Don't worry if you are not familiar with these terminologies, it's just saying that we are starting from a random line :math:`Y' = wX + b` , then we gradually adapt :math:`w` and :math:`b` to minimize the difference between :math:`Y'` and :math:`Y`. Here is what it looks like in PaddlePaddle:
```

Comment:
We use the neural network to learn this function mapping from :math:`X` to :math:`Y`,  This network has a  linear activation layer and a square error cost layer. Actually, it is just a line in two-dimensional space :math:`Y' = wX + b` . To learn the parameters, we gradually adjust the :math:`w` and :math:`b` to minimize the euclidean difference between math:`Y'` and :math:`Y`. Here is the related python code in PaddlePaddle: